benchmark/Streamly/Benchmark/Data/Array/Foreign/Mut.hs

```diff
@@ -38,7 +39,7 @@ sourceUnfoldrM :: (IsStream t, MonadAsync m) => Int -> Int -> t m Int
 sourceUnfoldrM value n = Stream.unfoldrM step n
     where
     step cnt =
-        if cnt > n + value
+        if cnt > n + (value - 1)
```

Comment:
why do we need -1 here?

Reply:
You added that, so you need to explain :-)

Reply:
ahh I thought initially array index generated by unfold has 0 to n causing indexoutofBound.
But it generates index 0 to n-1 so -1 is not needed.